ledger-service/http-json/src/main/scala/com/digitalasset/http/ContractsFetch.scala

```diff
@@ -382,9 +382,12 @@ private object ContractsFetch {
   final case class InsertDeleteStep[+C](inserts: Vector[C], deletes: Set[String]) {
     def append[CC >: C](o: InsertDeleteStep[CC])(
         implicit cid: CC <~< DBContract[Any, Any, Any]): InsertDeleteStep[CC] =
+      appendWithCid(o)(x => cid(x).contractId)
```

Comment:
Because working GADTs are too much to ask for

```suggestion
      appendWithCid(o)(Liskov.contra1_2[Function1, DBContract[Any, Any, Any], CC, String](cid)(_.contractId))
```